CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## [Unreleased]
+
+#### Warning - Potentially breaking change
+- Restivus used to store the account login token in the user document: services.resume.loginTokens.token
+- Restivus now stores the account login token as a hashed token, in the user document: services.resume.loginTokens.hashedToken
+- This matches Meteor Accounts package
+
+#### Fixed
+- Issue #79:
+  - Update to match standard Meteor login and Account token storage
```

Comment:
This should be moved to the "Changed" section. Fixes are for bugs. This was an intended feature at one time that just needed to be updated to match the current API. I recommend you change it to:

- Update default auth endpoints to match current Accounts token storage (see #79)